File: benchmark/results/java-invocation-inline/run-002/a-mcp/output/catalog/value_objects/Category.java
package com.metaengine.demo.catalog.value_objects;

import java.time.OffsetDateTime;

// Category value object.

public class Category {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public Category(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/ts-monolith-opus-inline/run-005/b-baseline/output/src/catalog/value-objects/Category.ts
import { Id } from '../../shared/value-objects/Id';
import { Timestamp } from '../../shared/value-objects/Timestamp';

/** Category value object. */
export interface Category {
  id: Id;
  createdAt: Timestamp;
  updatedAt: Timestamp;
  name: string;
  description: string;
}
